system/core-workflow/how-do-they-work.rst

```diff
@@ -0,0 +1,208 @@
+
+Available Operators
+^^^^^^^^^^^^^^^^^^^
+
+.. note::
+
+   The availability of available operators depends on the object type and scope.
+
+.. hint:: **🧐 Actions can cause confusion**
+
+   | Please note that actions may or may not restrict API based access to
+     attributes. We're displaying the following icons for your overview
+     to understand these limits better. 👀
+   | |api| This icon indicates the action affects the API.
+   | |ui| This Icon indicates the action only affects the web interface.
+
+show |ui|
+   Display the field in question. Allows setting of values.
+
+hide |ui|
+   Hide the field in question, however,
+   technically still allows setting the field.
+
+      .. warning::
+
+         The field is **not** gone and still contains any value it provides!
+         You may want to consider *remove* instead.
+
+remove |ui| |api|
+   Entirely removes the field. The field value cannot be changed / set.
+
+      .. warning::
+
+         This fields value is being unset in case it's set!
+         You may want to consider *hide* instead.
+
+set mandatory |ui| |api|
+   Sets the field to mandatory.
+
+set optional |ui| |api|
+   Sets the field to optional.
+
+add option |ui| |api|
+   Allows adding options to tree selects or selects.
+
+      .. note::
+
+         This requires options to be hidden before hand (remove option).
+         It allows to use *existing* configured values.
+
+remove option |ui| |api|
+   Allows removing options from tree selects or selects.
+
+      .. note::
+
+         It allows to use *existing* configured values.
+
+set fixed to |ui| |api|
+   Reduces the available options by your selection.
+
+      .. tip::
+
+         This may indirectly reduce your workflows in terms of
+         *add option* and *remove option*. 🤓
+
+fill in |ui|
+   Allows population of string and integer fields with your value.
+
+fill empty |ui|
+   Allows population of string and integer fields with your value
+   **if the field is empty**.   
+
+select |ui|
+   Select a specific value within a select, tree select or boolean fields.
+
+auto select |ui|
+   | Helps the user on tree selects and select fields:
+   | If the field has one option to select only and has no value yet, the
+     value is automatically set.
+
+   .. warning::
+
+      This option only works if you have one value and acts passive with more
+      options.
+
+set readonly |ui|
+   Allows you to display an attribute as read only.
+
+unset readonly |ui|
+   In case a workflow set the field in question to read only, you can
+   undo this with above option.
+
+.. |api| image:: /images/icons/api-symbol.png
+   :height: 42px
+   :width: 42px
+
+.. |ui| image:: /images/icons/ui-symbol.png
+   :height: 42px
+   :width: 42px
+
+Stop after match
+----------------
+
+Stop evaluation of other, following workflows that would match other wise.
+
+Default: ``no``
+
+Priority
+--------
+
+By adjusting the priority you decide when a workflow is checked on.
```

Comment:
maybe: "when a workflow is activated"?

Reply:
Doesn't fit in my opinion.
This part is about the user being able to adjust the priority of the workflows as needed.

So the order in which they are evaluated in.
While that's only relevant when they're active, the moment you adjust this value doesn't matter.

This screenshot may help (lower end)
https://github.com/zammad/zammad-admin-documentation/blob/mh/add/core-workflow/images/system/core-workflow/add-new-workflow.png

Reply:
Ah okay! That wasn't quite clear to me through the above wording, but perhaps that's just me. Perhaps we could rephrase to "you decide at which point the workflow is checked on", but I'll leave it up to you

Reply:
Okay, 
I switched this from `By adjusting the priority you decide when a workflow is checked on.`
to `You decide at which point your workflow is evaluated.` hoping it fits well.